src/SqlVirtualMachine/examples/New-AzSqlVM.md

```diff
@@ -20,3 +21,146 @@ Location	Name		ResourceGroupName
 eastus		sqlvm1		ResourceGroup01	
 ```
 
+Create a SQL virtual machine with specific sku type and license type.
+
+### Example 3
+```powershell
+New-AzSqlVM -ResourceGroupName 'ResourceGroup01' -Name 'sqlvm1' -Location 'eastus' -LicenseType 'AHUB'
+```
+
+```output
+Location	Name		ResourceGroupName
+--------	----		-----------------
+eastus		sqlvm1		ResourceGroup01	
+```
+
+Create a SQL virtual machine with AHUB billing tag.
+
+### Example 4
+```powershell
+New-AzSqlVM -ResourceGroupName 'ResourceGroup01' -Name 'sqlvm1' -Location 'eastus' -LicenseType 'DR'
+```
+
+```output
+Location	Name		ResourceGroupName
+--------	----		-----------------
+eastus		sqlvm1		ResourceGroup01	
+```
+
+Create a SQL virtual machine with DR billing tag.
+
+### Example 5
+```powershell
+New-AzSqlVM -ResourceGroupName 'ResourceGroup01' -Name 'sqlvm1' -Location 'eastus' -AdditionalFeatureServerConfigurationIsRServicesEnabled
+```
+
+```output
+Location	Name		ResourceGroupName
+--------	----		-----------------
+eastus		sqlvm1		ResourceGroup01	
+```
+
+Create a SQL virtual machine with Enable R services in SQL2016 onwards.
```

Comment:
Better to check with Amitabh/PM. We have stopped exposing this in portal so probably no needed in the example

Reply:
removed deprecated features from examples.